src/participant/participant.ts

```diff
@@ -287,10 +306,19 @@ export default class ParticipantController {
       connectionNames: this._getConnectionNames(),
     });
 
-    await this.streamChatResponseContentWithCodeActions({
-      modelInput,
-      token,
-      stream,
+    const { hasCodeBlock, outputLength } =
+      await this.streamChatResponseContentWithCodeActions({
+        modelInput,
+        token,
+        stream,
+      });
+
+    this._telemetryService.trackCopilotParticipantResponse({
+      command: 'generic',
+      has_cta: false,
```

Comment:
Same q as below, should this be true when we have a code block? There's the open in playground and run actions.